packages/ssm-cache/index.ts

```diff
@@ -46,31 +55,75 @@ export class SsmParameterNotFound extends Error {
   }
 }
 
-export const getSsmParameter = (name: string): string => {
-  if (!Object.prototype.hasOwnProperty.call(ssmCache.values, name)) {
-    throw new SsmParameterNotFound(`SSM parameter ${name} not found in cache`);
+export const hasParameterExpired = (parameter: SsmCacheParameter | undefined) => {
+  return !!(parameter?.expiryDate && new Date() > parameter.expiryDate);
+};
+
+export const hasCacheExpired = () => !!(ssmCache.expiryDate && new Date() > ssmCache.expiryDate);
+
+export const isConfigNotEmpty = () => !!Object.keys(ssmCache.values).length;
+
+export const setCacheDurationMilliseconds = (cacheDurationMilliseconds: number) => {
+  ssmCache.cacheDurationMilliseconds = cacheDurationMilliseconds;
+};
+
+export const parameterExistsInCache = (name: string) =>
+  Object.prototype.hasOwnProperty.call(ssmCache.values, name) && ssmCache.values[name];
```

Comment:
Is the first part of the check necessary?

Reply:
Good catch. This logic was flawed in several ways. Refactored to make it treat falsy values as missing as well.